kinto_algolia/indexer.py

```diff
@@ -23,7 +23,7 @@ def join(self):
         self.tasks = []
 
     def set_extra_headers(self, headers):
-        self.client.set_extra_headers(**headers)
+        self.client._config.headers.update(headers)
```

Comment:
That's the simplest way I found to not recreate the client each time.

Another solution would be to keep the application_id and api_key and recreate the client there.

Reply:
Sounds fine.